packages/kbn-i18n/src/core/i18n/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "browser": "./index.js",
```

Comment:
__nit:__ I'd still keep default `index.js` for node and `browser.js` for browser similar to what is used by npm itself: `main` for node and `browser` for browser.

Reply:
you are right, will fix it, thank you!